bulk

```diff
@@ -172,6 +197,7 @@ case $COMMAND in
   g|get) doGet;;
   l|list) doList;;
   e|export) doExport;;
+  s|status) doGetPublicationStatus;;
```

Comment:
LGTM but the options seem to be starting to drift from what this script can do.  Options in the usage could use the extra commands that were added.